.github/workflows/lint-test.yml

```diff
@@ -4,12 +4,30 @@ on:
   push:
   pull_request:
 
+
 permissions:
   contents: read
 
 jobs:
   build:
     runs-on: ubuntu-latest
+
+    services:
```

Comment:
@michplunkett @jamesturk @divij-sinha Is this the correct way to configure this file so that github actions runs the django pytests? I'm terrified to accidentally overwrite or delete our production database